app/controllers/api/services_controller.rb

```diff
@@ -108,31 +107,49 @@ def service_params
     params.require(:service).permit(permitted_params)
   end
 
+  # rubocop:disable Metrics/MethodLength
   def proxy_params
-    oidc_params = %i[oidc_issuer_type oidc_issuer_endpoint jwt_claim_with_client_id jwt_claim_with_client_id_type] + [{oidc_configuration_attributes: OIDCConfiguration::Config::FLOWS + [:id]}]
-    permitted_params = oidc_params + %i[
-      auth_user_key auth_app_id auth_app_key credentials_location hostname_rewrite secret_token
-      error_status_auth_failed error_headers_auth_failed error_auth_failed
-      error_status_auth_missing error_headers_auth_missing error_auth_missing
-      error_status_no_match error_headers_no_match error_no_match
-      error_status_limits_exceeded error_headers_limits_exceeded error_limits_exceeded
+    permitted_params = [{oidc_configuration_attributes: OIDCConfiguration::Config::FLOWS + [:id]}] + %i[
+      oidc_issuer_type
+      oidc_issuer_endpoint
+      jwt_claim_with_client_id
+      jwt_claim_with_client_id_type
+      auth_user_key
+      auth_app_id
+      auth_app_key
+      credentials_location
+      hostname_rewrite
+      secret_token
+      error_status_auth_failed
+      error_headers_auth_failed
+      error_auth_failed
+      error_status_auth_missing
+      error_headers_auth_missing
+      error_auth_missing
+      error_status_no_match
+      error_headers_no_match
+      error_no_match
+      error_status_limits_exceeded
+      error_headers_limits_exceeded
+      error_limits_exceeded
     ]
```

Comment:
This can be initialized in a constant outside from the method as DEFAULT_PARAMS.